src/System.Runtime.Analyzers/Core/SpecifyCultureInfo.cs

```diff
@@ -43,6 +43,8 @@ public override void Initialize(AnalysisContext analysisContext)
 
             analysisContext.RegisterCompilationStartAction(csaContext =>
             {
+                var obsoleteAttributeType = WellKnownTypes.ObsoleteAttribute(csaContext.Compilation);
```

Comment:
Yes, I am aware that `obsoleteAttributeType` can be null, and I believe this is already handled by the existing code. Since `attributeType` will be null, the LINQ query in `WhereMethodDoesNotContainAttribute` won't `.Equals()` any `AttributeClass`, therefore it will return all methods.